pkg/js/README.md

```diff
@@ -1,16 +1,28 @@
-Methods and annotations to specify interoperability with JavaScript APIs.
+Use this package when you want to call JavaScript APIs from Dart code, or vice versa.
 
-### Example
+This package's main library, `js`, provides annotations and functions
+that let you specify how your Dart code interoperates with JavaScript code.
+The Dart-to-JavaScript compilers — dartdevc and dart2js — recognize these
+annotations, using them to connect your Dart code with JavaScript.
+
+**Important:** This library supercedes `dart:js`, so don't import `dart:js`.
+Instead, import `package:js/js.dart`.
+
+A second library in this package, `js_util`, provides low-level utilities
+that you can use if you need to read property values.
```

Comment:
I would phrase this as something like "that you can use as a fallback where it is not possible to wrap with a static annotated API."